File: regolith/filters_data/system_template/educator_tools/feedback_prompt/_map.py
[
    {"source": "entities/*.geo.json", "target": AUTO_FLAT, "json_template": True},
    {"source": "entities/*.entity.png", "target": AUTO_FLAT},
    {"source": "entities/*.behavior.json", "target": AUTO_FLAT, "json_template": True},
    {"source": "entities/*.entity.json", "target": AUTO_FLAT, "json_template": True},
    {"source": "entities/*.rc.json", "target": AUTO_FLAT, "json_template": True},
    {"source": "dialogues/*.dialogue.json", "target": AUTO_FLAT, "json_template": True},
    {
        "source": "translations/*.lang",
        "subfunctions": True,
        "target": AUTO_FLAT,
        "on_conflict": "append_end",
    },
]

File: regolith/filters_data/modular_mc/educator_tools/feedback_prompt/_map.ts
import { scope } from "./scope.ts";

export const MAP = [
	{
		source: "entities/*.geo.json",
		target: ":autoFlat",
	},
	{
		source: "entities/*.entity.png",
		target: ":autoFlat",
	},
	{
		source: "entities/*.behavior.json",
		target: ":autoFlat",
	},
	{
		source: "entities/*.entity.json",
		target: ":autoFlat",
	},
	{
		source: "entities/*.rc.json",
		target: ":autoFlat",
	},
	{
		source: "dialogues/*.dialogue.json",
		target: ":autoFlat",
		jsonTemplate: true,
		scope: scope,
	},
	{
		source: "translations/*.lang",
		target: ":autoFlat",
		onConflict: "appendEnd",
	},
];
